definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -5157,5 +5157,26 @@
     "CaseEventID": "refertoLegalAdvisor",
     "UserRole": "caseworker-divorce-solicitor",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/03/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorApplyForDN",
+    "UserRole": "caseworker-divorce-solicitor",
+    "CRUD": "R"
+  },
+  {
```

Comment:
Add read-only permissions so any solicitor can see events in History tab

Reply:
ok, nevermind my prev question :)